### ISSUE TITLE
MD004 - give specific order and characters for sublist

### ISSUE THREAD

**Issue by skwde** [DavidAnson/markdownlint#413]
When we configure
```yaml
MD004:
    style: sublist
```
any order of `+`, `*` and `-` is allowed. 

How do I specify a specific order, e.g. `['-', '+', '*']`?
This should be applied cyclic, e.g. the 4th level list item should again be a `-`.
